### ISSUE TITLE
Crash on bookmarking an item from Explore

### ISSUE THREAD

**Issue by maskaravivek** [commons-app/apps-android-commons#2041]
**Summary:** 

Trying to bookmark an item from Explore > Media details activity crashes the app. 

**Steps to reproduce:** 

Explore > Media details > Bookmark item

How can we reproduce the issue? 
What did you expect the app to do, and what did you see instead?

**System logs:**

```
2018-11-27 23:15:56.987 32578-32578/fr.free.nrw.commons E/AndroidRuntime: FATAL EXCEPTION: main
    Process: fr.free.nrw.commons, PID: 32578
    java.lang.NullPointerException: Attempt to invoke virtual method 'java.lang.String fr.free.nrw.commons.bookmarks.Bookmark.getMediaName()' on a null object reference
        at fr.free.nrw.commons.bookmarks.pictures.BookmarkPicturesDao.findBookmark(BookmarkPicturesDao.java:129)
        at fr.free.nrw.commons.bookmarks.pictures.BookmarkPicturesDao.updateBookmark(BookmarkPicturesDao.java:70)
        at fr.free.nrw.commons.media.MediaDetailPagerFragment.onOptionsItemSelected(MediaDetailPagerFragment.java:161)
        at android.support.v4.app.Fragment.performOptionsItemSelected(Fragment.java:2476)
        at android.support.v4.app.FragmentManagerImpl.dispatchOptionsItemSelected(FragmentManager.java:3343)
        at android.support.v4.app.FragmentController.dispatchOptionsItemSelected(FragmentController.java:347)
        at android.support.v4.app.FragmentActivity.onMenuItemSelected(FragmentActivity.java:413)
        at android.support.v7.app.AppCompatActivity.onMenuItemSelected(AppCompatActivity.java:195)
        at android.support.v7.view.WindowCallbackWrapper.onMenuItemSelected(WindowCallbackWrapper.java:108)
        at android.support.v7.view.WindowCallbackWrapper.onMenuItemSelected(WindowCallbackWrapper.java:108)
        at android.support.v7.app.ToolbarActionBar$2.onMenuItemClick(ToolbarActionBar.java:63)
        at android.support.v7.widget.Toolbar$1.onMenuItemClick(Toolbar.java:203)
        at android.support.v7.widget.ActionMenuView$MenuBuilderCallback.onMenuItemSelected(ActionMenuView.java:780)
        at android.support.v7.view.menu.MenuBuilder.dispatchMenuItemSelected(MenuBuilder.java:822)
        at android.support.v7.view.menu.MenuItemImpl.invoke(MenuItemImpl.java:171)
        at android.support.v7.view.menu.MenuBuilder.performItemAction(MenuBuilder.java:973)
        at android.support.v7.view.menu.MenuBuilder.performItemAction(MenuBuilder.java:963)
        at android.support.v7.widget.ActionMenuView.invokeItem(ActionMenuView.java:624)
        at android.support.v7.view.menu.ActionMenuItemView.onClick(ActionMenuItemView.java:150)
        at android.view.View.performClick(View.java:6294)
        at android.view.View$PerformClick.run(View.java:24770)
        at android.os.Handler.handleCallback(Handler.java:790)
        at android.os.Handler.dispatchMessage(Handler.java:99)
        at android.os.Looper.loop(Looper.java:164)
        at android.app.ActivityThread.main(ActivityThread.java:6494)
        at java.lang.reflect.Method.invoke(Native Method)
        at com.android.internal.os.RuntimeInit$MethodAndArgsCaller.run(RuntimeInit.java:438)
        at com.android.internal.os.ZygoteInit.main(ZygoteInit.java:807)
```

**Device and Android version:** 
Nexus emulator. Android 8.1.
 
**Commons app version:** 

2.9.0 

**Would you like to work on the issue?**

Preferably not.

**Comment by sivaraam** [comment]
I could confirm this too. My device is Galaxy J1 Ace running Android 5.1.1.

**Comment by misaochan** [comment]
Seems to happen for everyone. Will fix ASAP.

**Comment by domdomegg** [comment]
Fixed in #2047
